ui/main/src/app/modules/feed/components/card-list/card-list.component.ts

```diff
@@ -118,4 +120,8 @@ export class CardListComponent implements AfterViewChecked, OnInit {
     declineAckAllCards(): void {
         this.modalRef.dismiss();
     }
+
+    isSelectedGroupedChildOfId(selected: string, id: string) {
```

Comment:
it takes me some time to undersand the behavior , proposal to rename it to isCardInGroup(cardId:string,groupParentCardId:string) ?

Reply:
Good suggestion. I renamed the method as suggested.